src/config/default.ts

```diff
@@ -15,5 +15,7 @@ export const defaultConfig: Config = {
   dbSSLValidate: false,
   dbName: "editor",
   editorS3Bucket: "editor",
-  srcS3Bucket: "kryia"
+  srcS3Bucket: "kryia",
```

Comment:
Something for a future follow up, but we should probably remove eLife specific terms from the defaults at some point.

Reply:
https://app.zenhub.com/workspaces/libero-editor-5d56a6cfaa13c900012ad75b/issues/libero/editor/801 should remove this value.